src/aiodynamo/errors.py

```diff
@@ -110,7 +115,12 @@ class PointInTimeRecoveryUnavailable(AIODynamoError):
 
 
 class TransactionCanceled(AIODynamoError):
-    pass
+    def __init__(self, body: Dict[str, Any]):
```

Comment:
could you add a type hint on the class for `cancellation_reasons` to make it clear that this is part of the public api.